web/init/src/components/shared/NavBar.jsx

```diff
@@ -183,9 +184,9 @@ export class NavBar extends React.Component {
     return (
       <div className={`NavBarWrapper flex flex-auto ${className || ""}`}>
         <div className="container flex flex1">
-          <div className="flex1 justifyContent--flexStart alignItems--center">
+          <div className="flex1 flex justifyContent--center alignItems--center">
             <div className="flex1 flex">
-              <div className="flex flex-auto">
+              <div className="flex flex-auto mteadata-wrapper">
```

Comment:
mteadata-wrapper => metadata-wrapper

Reply:
Good catch @vanpelt, will update in my next commit. Thanks!